flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: beb19de904b0159b454a839011b2f0f3
+// flow-typed version: <<STUB>>/babel-cli_v^6.18.0/flow_v0.37.4
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'babel-cli'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the 
+ * community by sending a pull request to: 
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'babel-cli' {
```

Comment:
Yes you can run `flow-typed install` and put it into the postinstall script. However I had some problems on my private projects with that. On some platforms it did not work. That's why I included it.